metadata.yaml

```diff
@@ -2,22 +2,19 @@
 # See LICENSE file for licensing details.
 
 name: pollen
-display-name: Charm Template
-summary: A very short one-line summary of the charm.
+display-name: Pollen
+summary: "Pollen Entropy-as-a-Service"
+maintainers:
+  - launchpad.net/~canonical-is-devops
+issues: https://github.com/canonical/pollen-operator/issues
+source: https://github.com/canonical/pollen-operator
+docs: Yet to be published.
 
 description: |
-  A single sentence that says what the charm is, concisely and memorably.
+  Pollen is a fast, secure, Entropy-as-a-Service server.
```

Comment:
The description doesn't comply with the documentation guidelines. If needed, check others charms as a reference